flowauth/backend/flowauth/token_management.py

```diff
@@ -226,3 +223,55 @@ def add_token(server):
     db.session.add(token)
     db.session.commit()
     return jsonify({"token": token_string, "id": token.id})
+
+
+def generate_token(
```

Comment:
I feel like you mentioned this, but why not use the `generate_token` function that's already present in the new module `flowkit_jwt_generator`?

Reply:
Docker build context scope.

Reply:
So we _could_ use it if we publish `flowkit_jwt_generator` on PyPI, for example?

Reply:
You'd need to publish to Pypi part way through the CI process, and would only be viable for tagged builds - in any other case, you're running the risk that flowkit_jwt_generator has changed in that branch.

Reply:
Good point. I suppose there are workarounds (e.g. copying the `flowkit_jwt_generator/` folder into `flowauth/` before building, and aborting the build if it is empty) but certainly not worth the hassle for a small function like this.